cmd/uplink/cmd/meta_get.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/internal/fpath"
+	"storj.io/storj/pkg/process"
+)
+
+func init() {
+	addCmd(&cobra.Command{
+		Use:   "get [KEY] PATH",
+		Short: "Get a Storj object's metadata",
+		RunE:  metaGetMain,
+	}, metaCmd)
+}
+
+// metaGetMain is the function executed when metaGetCmd is called.
+func metaGetMain(cmd *cobra.Command, args []string) (err error) {
+	var key *string
+	var path string
+
+	switch len(args) {
+	case 0:
+		return fmt.Errorf("No object specified")
+	case 1:
+		path = args[0]
+	case 2:
+		key = &args[0]
+		path = args[1]
+	default:
+		return fmt.Errorf("Too many arguments")
+	}
+
+	ctx, _ := process.Ctx(cmd)
+
+	src, err := fpath.New(path)
+	if err != nil {
+		return err
+	}
+	if src.IsLocal() {
+		return fmt.Errorf("The source destination must be a Storj URL")
+	}
+
+	project, bucket, err := cfg.GetProjectAndBucket(ctx, src.Bucket())
+	if err != nil {
+		return err
+	}
+	defer closeProjectAndBucket(project, bucket)
+
+	object, err := bucket.OpenObject(ctx, src.Path())
+	if err != nil {
+		return err
+	}
+	defer object.Close()
+
+	if key != nil {
+		var keyNorm string
+		err := json.Unmarshal([]byte("\""+*key+"\""), &keyNorm)
+		if err != nil {
+			return err
+		}
+
+		value, ok := object.Meta.Metadata[keyNorm]
+		if ok != true {
+			return fmt.Errorf("Key does not exist")
+		}
+
+		str, err := json.Marshal(value)
+		if err != nil {
+			return err
+		}
+
+		fmt.Printf("%s\n", str[1:len(str)-1])
```

Comment:
I wonder whether we should keep the quotes to keep things consistent with always printing valid json? (kindof)

Reply:
From a shell the UX might not be as nice:

```
meta set 'some json string' 'sj://bucket/file'
meta set "$variable" 'sj://bucket/file'
```

vs

```
meta set '"some json string"' 'sj://bucket/file'
meta set "\"some json string\"" 'sj://bucket/file'
meta set "\"$variable\"" 'sj://bucket/file'
```

Reply:
(Assuming we wanted symmetry with the `meta set` that used to exist and may one day again.)

Reply:
symmetry is nice :)